PowerEditor/src/Notepad_plus.cpp

```diff
@@ -795,10 +795,16 @@ LRESULT Notepad_plus::init(HWND hwnd)
 
 	DockingManagerData& dmd = nppGUI._dockingData;
 
+	// preset minimal panel dimensions according to the current DPI
+	dmd._minDockedPanelVisibility = DPIManagerV2::scale(nppGUI._dockingData._minDockedPanelVisibility, dpi);
+	dmd._minFloatingPanelSize.cy = nppGUI._dockingData._minDockedPanelVisibility;
+	dmd._minFloatingPanelSize.cx = std::max(static_cast<int>(nppGUI._dockingData._minFloatingPanelSize.cy * 6),
+		::GetSystemMetrics(SM_CXMINTRACK));

```

Comment:
@xomx 
could you replace `GetSystemMetrics` with https://github.com/notepad-plus-plus/notepad-plus-plus/blob/e7e88a380ac01f8def627f1672f0192461603d6b/PowerEditor/src/dpiManagerV2.h#L49

Reply:
@ozone10 
Done.